outputs/html/default-template.hbs

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <title>OSS Attribution</title>
+  <style>
+    pre {
+      white-space: pre-wrap;
+      background: #eee;
+      padding: 24px;
+    }
+  </style>
+</head>
+
+<body>
+  <h1>OSS Attribution</h1>
+  <ol>
+    {{#buckets}}
+    {{#packages}}
+    <li>
+      <details>
+        <summary>{{name}} {{version}} - {{../name}}</summary>
+        {{#if website}}
+        <p><a href="{{website}}">{{website}}</a></p>
+        {{/if}}
+        {{#if copyrights}}
+        <ul>
+          {{#copyrights}}
+          <li>{{this}}</li>
+          {{/copyrights}}
+        </ul>
+        {{/if}}
+        <pre>{{../text}}</pre>
```

Comment:
The license text is identical for each block, but is repeated for each package. Since this is a template it doesn't matter much (anyone can customize to their needs) but I thought I'd give you a heads up.

Reply:
Right. we are actually flattening the buckets on our end and repeating the license text for each package so I just left it that way.

It might make sense to demonstrate the buckets more in the default template to show off the functionality though